All_home_works/hw9/task3.py

```diff
@@ -41,5 +42,5 @@ def count_lines(file):
 def get_files_from_directory(dir_path, file_extension):  # noqa: CCR001
     for dirpath, _, filenames in os.walk(dir_path):
         for file in filenames:
-            if file.split(".")[-1] == file_extension:
+            if file.endswith(f".{file_extension}"):
```

Comment:
Let's find one more way to work with extensions.

Reply:
`if pathlib.Path(file).suffix == f'.{file_extension}':`
`if os.path.splitext(file)[-1] == f'.{file_extension}':`